strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NugetArtifactGenerator.java

```diff
@@ -172,38 +171,33 @@ public void generate(Path packagePath,
     public void createArchive(Nuspec nuspec,
                               Path packagePath,
                               long bytesSize)
-            throws IOException,
-                   JAXBException,
-                   NoSuchAlgorithmException
+            throws IOException, JAXBException, NoSuchAlgorithmException
```

Comment:
```suggestion
            throws IOException,
                   JAXBException,
                   NoSuchAlgorithmException
```

Reply:
@luan-cestari : I don't know what your IDE's settings are like, or whether you're pulling before force pushing, but you're reverting my code changes.